addon/components/layers/wms-esri-layer.js

```diff
@@ -71,13 +71,9 @@ export default WMSLayerComponent.extend({
     this._super(...arguments);
 
     let innerEsriDynamicLayerProperties = {
-      leafletMap: this.get('leafletMap'),
-      leafletContainer: this.get('leafletContainer'),
       layerModel: this.get('layerModel'),
       crs: this.get('crs'),
-      index: this.get('index'),
-      visibility: this.get('visibility'),
-      restUrl: this.get('restUrl'),
+      url: this.get('url'),
```

Comment:
so it must be "url: this.get('restUrl')" and such a property should be defined in wms layer?